build-sensor/include/step1.h

```diff
@@ -0,0 +1,16 @@
+
+class step1 {
+    public:
+        void setup() {
+            // Configure mode of LED port
```

Comment:
should there be a blank line above this?
```suggestion

            // Configure mode of LED port
```

Reply:
Thanks, Done!